ktor-server/ktor-server-host-common/jvm/src/io/ktor/server/engine/StaticApplicationEngineEnvironment.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.engine
+
+import io.ktor.events.*
+import io.ktor.events.EventDefinition
+import io.ktor.server.application.*
+import io.ktor.server.config.*
+import io.ktor.util.logging.*
+import kotlin.coroutines.*
+
+internal class StaticApplicationEngineEnvironment(
```

Comment:
looks like it can be shared with nix implementation, except ClassLoader I think. But still then expect/actual can be just cleaner IMO